compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -3746,4 +3746,4 @@ THRIFT_REGISTER_GENERATOR(go, "Go",
                           "    read_write_private\n"
                           "                     Make read/write methods private, default is public Read/Write\n" \
                           "    use_context\n"
-                          "                     Make service method receive a context as first argument.\n")
+                          "                     Make service method receive a context as first argument, only go1.7+ is supported.\n")
```

Comment:
I don't think we should commit it. Right now, we don't have a `vendor` folder [in the library](https://github.com/apache/thrift/tree/master/lib/go). When people use `go get` (or other dependency tools) they'll get the `x/net/context` dependency automatically (if they are on Go<1.7).

What do you think?